front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -344,15 +344,15 @@ async function handler(
         vault
       );
 
-      // For managed data source, we create a default view in the workspace vault.
-      if (dataSource.isManaged()) {
-        const globalVault = await VaultResource.fetchWorkspaceGlobalVault(auth);
+      // For all data sources, we create a default view in the workspace vault.
```

Comment:
looking at the file name & the code, isn't it `all` => `managed` & `workspace vault` => `global vault` ?